src/modules/Builder/pages/BuilderApplet/BuilderApplet.utils.tsx

```diff
@@ -981,6 +985,8 @@ export const getDefaultValues = (appletData?: SingleApplet, defaultThemeId?: str
       : [],
     activityFlows: getActivityFlows(appletData.activityFlows, appletData.activities),
     streamEnabled: !!appletData.streamEnabled,
+    lorisIntegration: hasLorisIntegration,
```

Comment:
I suppose 'lorisIntegration' spreads redundant property because it can be figured out from 'integrations' in any place where it needs